queue/queue.py

```diff
@@ -13,16 +13,44 @@
 Stretch: What if you could only use instances of your Stack class to implement the Queue?
          What would that look like? How many Stacks would you need? Try it!
 """
-class Queue:
+
+import sys
+
+sys.path.append('./singly_linked_list/')
+from singly_linked_list.singly_linked_list import Node, LinkedList
+
+class ArrQueue:
+    def __init__(self):
+        self.storage = []
+    
+    def __len__(self):
+        return len(self.storage)
+
+    def enqueue(self, value):
+        self.storage.append(value)
+
+    def dequeue(self):
+        if len(self.storage) == 0:
+            return None
+        else:
+            return self.storage.unshift()
```

Comment:
Everything for the ArrQueue class is implemented perfectly! 

This last line in dequeue, you just need the right list method to remove the first element from the list .

In this case `self.storage.pop(0)` will work. `pop()` without any index provided will remove the last element, passing in `0` as the argument will remove the first element.

With this change all your tests for ArrQueue are passing!